lib/linkerd2-trace-context/src/propagation.rs

```diff
@@ -85,11 +84,10 @@ fn unpack_grpc_trace_context<B>(request: &http::Request<B>) -> Option<TraceConte
 fn parse_grpc_trace_context_fields(buf: &mut Bytes) -> Option<TraceContext> {
     trace!(message = "reading binary trace context", ?buf);
 
-    let version = try_split_to(buf, 1).ok()?;
+    let _version = try_split_to(buf, 1).ok()?;
```

Comment:
Nit: do we not need to propagate the trace context version? It would be helpful to have a comment here explaining why we just ignore this...